docs/src/pages/system/basics/basics.md

```diff
@@ -200,7 +200,7 @@ Cons:
   | c. Render 1,000 styled components | `<StyledDiv>`         | 160ms           |
   | d. Render 1,000 Box               | `<Box sx={…}>`        | 370ms           |
 
-  _Head to the [benchmark folder](https://github.com/mui-org/material-ui/tree/next/benchmark/browser) for a reproduction of these metrics._
+  _Head to the [benchmark folder](https://github.com/mui-org/material-ui/tree/master/benchmark/browser) for a reproduction of these metrics._
```

Comment:
```suggestion
<!-- #default-branch-switch -->

  _Head to the [benchmark folder](https://github.com/mui-org/material-ui/tree/master/benchmark/browser) for a reproduction of these metrics._
```

Adding the `<!-- #default-branch-switch -->` so that we don't forget again in the future :)